.github/workflows/maven.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         cache: maven
 
     - name: Build
-      run: mvn -B package --file pom.xml
+      run: mvn -Dhttp-client.idle-timeout=2m -B package --file pom.xml
```

Comment:
Did the CI hang or is some lib slow to download?

Reply:
Yeah - we've seen this GH runners be very slow sometimes so I added this.